### ISSUE TITLE
Email Helper is nonexistent on /system/helpers

### ISSUE THREAD

**Issue by valsopi** [pocketarc/codeigniter#16]
I just noticed that the 'email_helper.php' is not on /system/helpers. Is there any reason behind this?

**Comment by pocketarc** [comment]
Yes! This fork is based on CodeIgniter 3.2, where the email helper was removed (it was deprecated; 3.2 just removed it).

> The entire *Email Helper*, which only had two functions:
> - `valid_email()` (use `filter_var($email, FILTER_VALIDATE_EMAIL)` instead)
> - `send_email()` (use `mail()` instead)

https://github.com/bcit-ci/CodeIgniter/blob/develop/user_guide_src/source/installation/upgrade_320.rst

**Comment by pocketarc** [comment]
This helper has been added back to the repository to reduce backward compatibility issues.
